returnn/datasets/util/feature_extraction.py

```diff
@@ -199,6 +199,10 @@ def get_audio_features(self, audio, sample_rate, seq_name=None):
         feature_data = _get_audio_db_mel_filterbank(**kwargs)
       elif self.features == "linear_spectrogram":
         feature_data = _get_audio_linear_spectrogram(**kwargs)
+      elif self.features == "f_0":
```

Comment:
`f_0` is inconsistent to `f0` as used elsewhere. I would suggest here:
```suggestion
      elif self.features == "f0":
```